### DISCUSSION TITLE
Are "hide-" tags supposed to be honored?

### DISCUSSION THREAD

**Discussion by tobydriscoll** [jupyter-book/mystmd#1706] [Q&A] [answered]
I can't see where the cell tags honored by myst are documented. However, the remove-input, remove-output, and remove-cell tags seem to work as I expect. But the hide-input, hide-output, and hide-cell tags are ignored in my simple tests. Is the functionality gone or renamed, or am I doing something wrong? 

E.g.,

`````
## remove output
```{code-cell}
:tags: remove-output
print("Hello, world!")
```

## remove input
```{code-cell}
:tags: remove-input
print("Hello, world!")
```

## hide input
```{code-cell}
:tags: hide-input
print("Hello, world!")
```

## hide output
```{code-cell}
:tags: hide-output
print("Hello, world!")
```

## hide cell
```{code-cell}
:tags: hide-cell
print("Hello, world!")
```

`````

**Comment by agoose77** [comment]
In mystmd these are currently not implemented but are on our Jupyter book beta todo list!

**Comment by choldgraf** [comment, marked as answer]
Here's the issue tracking hide support via tags in jupyter book 2 / myst:

-  https://github.com/jupyter-book/mystmd/issues/1108
